src/React/Widgets/NumberSliderWidget/index.js

```diff
@@ -62,14 +62,14 @@ export default React.createClass({
           className={style.range}
           value={this.props.value}
           onChange={this.valInput}
-          max={max} min={min}
+          max={max} min={min} step={this.props.step || 'any'}
         />
         <input
           type="number"
           className={style.text}
           value={this.props.value}
           onChange={this.valInput}
-          max={max} min={min}
+          max={max} min={min} step="any"
```

Comment:
Neat, I didn't know about the 'any' option for `<input step>`. Shouldn't it be the same as the slider though? `this.props.step || 'any'`?